telosnetwork/telos-wallet
# adding receive page fully functional

# Fixes #177
This PR includes a fully functional receive page that generates a QR for the logged account address and provides a copy button to copy the address to the clipboard.

## Test scenarios
- go to [this link](https://deploy-preview-266--wallet-staging.netlify.app/)
- login in EVM
- press receive button on top header
  - you should see the QR dislpaying your address
  - you should see your address at the bottom of the page
  - if you are in desktop, you should see a little copy icon at the right of the address
  - if you are in mobile, you should see a copy button at the bottom of the address
  - if the address do not enter in one line it should break into several lines instead of going out of bounds
- press the copy button or icon.
  - it should copy the address to your clipboard
  - a success notification should appear

## update
- when you enter the receive page
  - it displays the back button on the header space
  - it hides the menu icon and the user info on the header
  - it displays using animation from left to right
- the QR size now adapts on small screens

## Screenshots

Desktop:
![image](https://user-images.githubusercontent.com/4420760/233874515-f36aa9c7-9d19-47ee-b060-9f5ca911707a.png)
Mobile:
![image](https://user-images.githubusercontent.com/4420760/233874187-0bde69d0-96d7-45c2-9321-a91f86146d1e.png)




## Checklist:
